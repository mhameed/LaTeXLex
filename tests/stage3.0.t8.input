 \phi_M(y)=1 for all y \in {\cal L}_{yes}, and
\medskip

 \phi_M(y)=0 for all y \in {\overline I} \setminus {\cal L}_{yes}.
\medskip

(Function \phi_M(y) was defined previously and carries a halting (for M)
input y to the corresponding output computed by M.) 

