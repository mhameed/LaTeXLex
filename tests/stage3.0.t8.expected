 \phi_M(y)=1 for all y \in {\cal L}_{yes}, and

\phi_M(y)=0 for all y \in {\overline I} \setminus {\cal L}_{yes}.

(Function \phi_M(y) was defined previously and carries a halting (for M)
input y to the corresponding output computed by M.) 


